src/main/kotlin/org/dashevo/dpp/util/JsonUtils.kt

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Team
+ *
+ * This source code is licensed under the MIT license found in the
+ * COPYING file in the root directory of this source tree.
+ */
+package org.dashevo.dpp.util
+
+import java.util.ArrayList
+import java.util.HashMap
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+
+object JsonUtils {
```

Comment:
Any reason we're not using `JSONObject.toMap()` and `JSONArray.toList()`?

Reply:
This code came from: https://stackoverflow.com/questions/21720759/convert-a-json-string-to-a-hashmap

I didn't know about `toMap` and `toList` -- will look into these.

Reply:
Wow, this article is 6 years old.

Reply:
This is resolved here:  e9f2ac5

JsonUtils was removed and replaced with `JsonObject.toMap()`